portafly/src/tests/components/AppLayout.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { render, fireEvent } from '@test/setup'
+import { AppLayout, IAppLayoutProps } from '@src'
+
+function makeAppLayout(
+  props: Partial<React.PropsWithChildren<IAppLayoutProps>> = {}
+) {
+  props = Object.assign(
+    {
+      logo: <div>App logo</div>,
+      avatar: <div>Avatar</div>,
+      toolbar: <div>Toolbar</div>,
+      navVariant: 'vertical',
+      navItems: [
+        {
+          title: 'Samples',
+          to: '/samples/',
+          items: [
+            { to: '/samples/foo', title: 'Foo' },
+            undefined,
+            { to: '/samples/bar', title: 'Bar' },
+            { to: '/samples/baz', title: 'Baz' },
+          ],
+        },
+        { to: '/support', title: 'Support' },
+        { to: '/something', title: 'Something' },
+      ],
+      navGroupsStyle: 'expandable',
+      startWithOpenNav: true,
+      theme: 'dark',
+      mainContainerId: 'test-main-container',
+      children: <div data-testid={'test-content'}>test</div>,
+    },
+    props
+  )
+  return <AppLayout {...(props as IAppLayoutProps)} />
+}
+
+function renderAppLayout(...args: Parameters<typeof makeAppLayout>) {
+  return render(makeAppLayout(...args))
+}
+
+describe('AppLayout tests', () => {
+  test('should render as configured', async () => {
+    const { getByTestId, getByText, container } = renderAppLayout()
+    getByTestId('test-content')
```

Comment:
Why are we not using snapshots?

Reply:
@josemigallas As I stated before, all the layout components were extracted from Fuse's lib, and used to bootstrap this. I'll ignore any further comment on them. We could open PRs in the future to enhance them.

Reply:
Sorry you're right, I keep forgetting about this